README.md

```diff
@@ -464,6 +464,12 @@ For complex products, like technology platforms, it's a best practice for the de
 - [Communities aren't funnels](https://www.youtube.com/watch?v=EDgNuFdVay0) — Josh Dzielak, DevRelCon 2019
 - [How we're using Orbit to build Orbit](https://orbit.love/blog/how-we-use-orbit-to-build-orbit/) - Orbit Blog
 
+<p align="left">
```

Comment:
@alexlsalt Couple of suggestions for the section:

- Give it a sub-heading "Orbit chat icons"
- Graphic underneath (that's a nice touch!)
- Clicking the graphic downloads the zip file (you can add the zip file to the repo)
- Underneath that, two links: Download the icons (zip file) and Read the blog post (blog post link)

The intent there is to make the clickthrough behavior more obvious and provide a direct download to the zip file. The section sub-header might help it stand out, see what you think about that.

Reply:
Awesome - thanks for these suggestions! Made the changes 👍